runner/run_benchmark.py

```diff
@@ -12,23 +11,13 @@
 from pathlib import Path
 from dataset_interfaces.factory import DatasetFactory, DATASETS
 from dataset_interfaces.interface import TestExample
-from model_interfaces.length_bias_agent import LengthBiasAgent
 from model_interfaces.interface import ChatSession
 from model_interfaces.llm_interface import LLMChatSession, TimestampLLMChatSession
-from model_interfaces.ltm_agent_wrapper import LTMAgentWrapper
-from model_interfaces.cost_estimation import CostEstimationChatSession
-from model_interfaces.human import HumanChatSession
-from model_interfaces.huggingface_interface import HFChatSession
-from model_interfaces.gemini_interface import GeminiProInterface
-from model_interfaces.memorybank_interface import MemoryBankInterface #Memory Bank
-from model_interfaces.memgpt_interface import MemGPTInterface
```

Comment:
Why the change to local imports?

Reply:
It all started with MemGPT, and when I couldn't get around its dependencies. Then I realized that the same could happen with MemoryBank or any of those 3rd party agents. With the local imports we could let the user to install only what they're actually using. What do you think?